media-gfx/gnofract4d/gnofract4d-4.3_p20200821.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+DISTUTILS_SINGLE_IMPL=1
+
+inherit distutils-r1 eutils virtualx xdg
+
+COMMIT="47752e0d240d9f2686c2a69d813ac5112bda1ad3"
+
+DESCRIPTION="A program for drawing beautiful mathematically-based images known as fractals"
+HOMEPAGE="https://fract4d.github.io/gnofract4d/"
+SRC_URI="https://github.com/fract4d/gnofract4d/archive/${COMMIT}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+COMMON_DEPEND="
+	media-libs/libpng:0=
+	virtual/jpeg:0"
+RDEPEND="${COMMON_DEPEND}
+	$(python_gen_cond_dep '
+		dev-python/pycairo[${PYTHON_USEDEP}]
+		dev-python/pygobject:3[${PYTHON_USEDEP}]
+	')
+	x11-libs/gtk+:3[introspection]"
+BDEPEND="virtual/pkgconfig"
+DEPEND="${COMMON_DEPEND}"
```

Comment:
I think we can just drop COMMON_DEPEND and reorder these by setting RDEPEND to be DEPEND plus some bits.

Reply:
Done.